Contributing.md

```diff
@@ -0,0 +1,99 @@
+# Contributing to RealtimeSanitizer
+
+This document describes some helpful tips and tricks for contributing to RealtimeSanitizer. 
+
+RealtimeSanitizer is a part of LLVM, meaning that when in conflict, the two following guides take precedence over this one:
+
+1. [Contributing to LLVM](https://llvm.org/docs/Contributing.html)
```

Comment:
This site has been really helpful for the set-up process too! https://llvm.org/docs/GettingStarted.html

Reply:
Excellent suggestion, added!!